src/js/yeldaWebchatInjector.js

```diff
@@ -582,9 +583,14 @@ class YeldaChat {
     // default container
     this.parentContainer = document.body
     this.bubbleContainer = null
+    this.bubbleContainerClone = null
 
     if (data.bubbleContainerChildId && document.getElementById(data.bubbleContainerChildId)) {
       this.bubbleContainer = document.getElementById(data.bubbleContainerChildId).parentElement
+
+      if (this.bubbleContainer) {
+        this.bubbleContainerClone = document.getElementById(data.bubbleContainerChildId).parentElement.cloneNode(true)
+      }
     }
```

Comment:
can you comment that is was clone to be restored if the webchat is unloaded ?
thanks !